app/delegate-server/Const.purs

```diff
@@ -0,0 +1,23 @@
+module DelegateServer.Const
+  ( AppConst
+  , appConst
+  ) where
+
+import JS.BigInt (BigInt)
+import JS.BigInt (fromInt) as BigInt
+import Node.Path (FilePath)
+
+type AppConst =
+  { testnetMagic :: String
+  , protocolParams :: FilePath
+  , hydraScriptsTxHash :: String
```

Comment:
Sorry, what is this? Is it the hash for Hydra version of the standing bit validator?

Reply:
It is the hash of the transaction containing Hydra reference scripts in its outputs. This hash is published in release notes for hydra-node: https://github.com/input-output-hk/hydra/releases.